modules/op-release-notes-1-9.adoc

```diff
@@ -0,0 +1,138 @@
+// https://github.com/tektoncd/cli/pull/1684
+// Piyush Garg @piyush-garg
+
+* Before this update, to access bundle commands like `list` and `push`, it was required to use the `kubeconfig` file . With this update, this issue has been resolved and the `kubeconfig` file is not required to access bundle commands.
+// https://github.com/tektoncd/cli/pull/1557
+// Piyush Garg @piyush-garg
+
+* Before this update, if the parent pipelinerun is running while deleting TaskRuns, then TaskRuns will be deleted. With this update, this issue is resolved and TaskRuns are not getting deleted if the parent PipelineRun is running.
+// https://github.com/tektoncd/cli/pull/1736
+// Piyush Garg @piyush-garg
+
+* Before this update, if the user attempted to build a bundle with more objects than the pipeline controller permitted, the Tekton CLI did not display an error message. With this update, this issue is resolved and the Tekton CLI displays an error message if the user attempted to build a bundle with more objects than the limit permitted in the pipeline controller.
+// https://github.com/tektoncd/cli/pull/1572
+// Piyush Garg @piyush-garg
+
+// .Operator
+
+* Before this update, if namespaces were removed from the cluster, then the operator did not remove namespaces from the `ClusterInterceptor ClusterRoleBinding` subjects. With this update, this issue has been resolved, and the operator removes the namespaces from the `ClusterInterceptor ClusterRoleBinding` subjects.
+// Shubham Minglani
+
+* Before this update, the default installation of the Red Hat OpenShift Pipelines Operator resulted in the `pipelines-scc-rolebinding security context constraint` (SCC) role binding resource remaining in the cluster. With this update, the default installation of the Red Hat OpenShift Pipelines Operator results in the `pipelines-scc-rolebinding security context constraint` (SCC) role binding resource resource to be removed from the cluster.
+// https://github.com/tektoncd/operator/pull/1156
+// https://issues.redhat.com/browse/SRVKP-2520
+// Shubham Minglani
+
+// .PAC
+
+* Before this update, {pac} did not get updated values from the {pac} configmap. With this update, this issue is fixed and {pac} configmap looks for any new changes.
+// Savita Ashture
+
+* Before this update, {pac} controller did not wait for the `tekton.dev/pipeline` label to be updated and added the `checkrun id` label, which would cause race conditions. With this update, a fix has been added to the {pac} controller to wait for the `tekton.dev/pipeline` label to be updated and then add the `checkrun id` label, which helps to avoid race conditions.
+// Savita Ashture
+
+* Before this update, the `tkn-pac create repo` command did not override a `PipelineRun` if it already existed in the Git Repository. With this update, `tkn-pac create` command is fixed to override a `PipelineRun` if it exists in the Git Repository and this resolves the issue successfully.
```

Comment:
I am not sure if Git Repository should be upper case - in general, it should not, but maybe the conventions here are different

Reply:
Fixed.